src/bilder/images/edxapp/templates/edxapp/mitx/lms_only.yml

```diff
@@ -0,0 +1,118 @@
+ACCOUNT_MICROFRONTEND_URL: null
+ACE_CHANNEL_DEFAULT_EMAIL: django_email
+ACE_CHANNEL_TRANSACTIONAL_EMAIL: django_email
+ACE_ENABLED_CHANNELS:
+- django_email
+ACE_ENABLED_POLICIES:
+- bulk_email_optout
+ACE_ROUTING_KEY: edx.lms.core.default
+# ALTERNATE_WORKER_QUEUES: cms  # The code already has this default
+# ANALYTICS_API_KEY: ''
+# ANALYTICS_API_URL: http://localhost:18100
+API_ACCESS_FROM_EMAIL: api-requests@example.com
+API_ACCESS_MANAGER_EMAIL: api-access@example.com
+API_DOCUMENTATION_URL: http://course-catalog-api-guide.readthedocs.io/en/latest/
+AUDIT_CERT_CUTOFF_DATE: null
+AUTH_DOCUMENTATION_URL: http://course-catalog-api-guide.readthedocs.io/en/latest/authentication/index.html
```

Comment:
Probably not, these are just default values that were generated from the Ansible run that I used to baseline the configs and haven't revisited it since then.